### DISCUSSION TITLE
Multi select Dropdown selections visible

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [microsoft/fluentui#22606] [Q&A] [answered]
Does anyone know how to make the selected options to show each on new line?
Instead of: Option1, Option2, Option3 
Like this:
Option1 
Option2
Option3

My users are asking for it.

**Comment by micahgodbolt** [comment, marked as answer]
The options are rendered as a single span with items seperated by commas, so there isn't a simple way to style them to look stacked. You can probably use `onRenderTitle` to customize that content.
